sdks/java/extensions/euphoria/euphoria-beam/src/test/java/org/apache/beam/sdk/extensions/euphoria/beam/OperatorsTestSuite.java

```diff
@@ -15,23 +15,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.extensions.euphoria.operator.test;
+package org.apache.beam.sdk.extensions.euphoria.beam;
 
 import java.time.Duration;
-import org.apache.beam.sdk.extensions.euphoria.beam.BeamExecutor;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.FlatMapTest;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.JoinTest;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.ReduceByKeyTest;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.UnionTest;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.junit.ExecutorEnvironment;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.junit.ExecutorProvider;
+import org.apache.beam.sdk.extensions.euphoria.beam.testkit.junit.ExecutorProviderRunner;
 import org.apache.beam.sdk.extensions.euphoria.core.executor.Executor;
-import org.apache.beam.sdk.extensions.euphoria.operator.test.junit.ExecutorEnvironment;
-import org.apache.beam.sdk.extensions.euphoria.operator.test.junit.ExecutorProvider;
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
 
 /**
- * Local {@link ExecutorProvider}.
+ * Euphoria operators test suite.
  */
-public interface BeamExecutorProvider extends ExecutorProvider {
+@RunWith(ExecutorProviderRunner.class)
+@Suite.SuiteClasses({
+    //    BroadcastHashJoinTest.class,
+    //    CountByKeyTest.class,
+    //    DistinctTest.class,
+    //    FilterTest.class,
+    FlatMapTest.class,
+    JoinTest.class,
+    //    JoinWindowEnforcementTest.class,
+    //    MapElementsTest.class,
+    ReduceByKeyTest.class,
+    //    ReduceStateByKeyTest.class,
+    //    SumByKeyTest.class,
+    //    TopPerKeyTest.class,
```

Comment:
can we try to uncomment more tests? I'd say that MapElementsTest, DistinctTest, CountByKeyTest, SumByKeyTest, TopPerKeyTest should work.

Reply:
and FilterTest

Reply:
I will definitely try to do that. I believe that there will be some changes necessary to make these work, recalling the hassle we had last time with the operator testkit. I will see how it goes. New issue will be created if needed.